tests/functional/ContentCest.php

```diff
@@ -426,4 +426,62 @@ public function canSeeStDataMarkupWithFirstImageUrlFromTeaser(FunctionalTester $
             "height": "auto"
         }', 'script[type="application/ld+json"]');
     }
+
+    public function canSeeStDataMarkupWithItemListTypeOnCategoryPage(FunctionalTester $I)
+    {
+        $user = factory(User::class)->create();
+        $language = new Language(['code' => 'en']);
+
+        $root = dispatch_now(CreateContent::category('Parent - Title', $language, $user, [
+            'teaser'       => 'Parent - Title',
+            'published_at' => Carbon::now(),
+            'is_active'    => true
+        ]));
+        dispatch_now(CreateContent::category('Child 1 - Title', $language, $user, [
+            'teaser'       => 'Child 1 - Title',
+            'parent_id'    => $root->id,
+            'published_at' => Carbon::now(),
+            'is_active'    => true
+        ]));
+        dispatch_now(CreateContent::content('Child 2 - Title', $language, $user, [
+            'Child'        => 'Child 2 teaser',
+            'parent_id'    => $root->id,
+            'published_at' => Carbon::now(),
+            'is_active'    => true
+        ]));
+        dispatch_now(CreateContent::content('Child 3 - Title', $language, $user, [
+            'Child'        => 'Child 3 teaser',
+            'parent_id'    => $root->id,
+            'published_at' => Carbon::now(),
+            'is_active'    => true
+        ]));
+        dispatch_now(CreateContent::content('Child not published - Title', $language, $user, [
+            'Child'        => 'Child not published - Title',
+            'parent_id'    => $root->id,
+            'published_at' => Carbon::now()->addDay(),
+            'is_active'    => true
+        ]));
+
+        $I->amOnPage('parent-title');
+
+        $tag = 'script[type="application/ld+json"]';
+
+        $I->see('"@type":"ItemList"', $tag);
+        $I->see('"itemListElement":{
+            "@type": "ListItem",
+            "position": 1,
+            "url":"http://dev.gzero.pl/parent-title/child-1-title"
```

Comment:
Please use some helper e.g url_base for the first part to make sure that this test will work for a different domain.